packages/helpers/src/builders/poll.ts

```diff
@@ -0,0 +1,63 @@
+import type { Poll, PollLayoutType } from "lilybird";
+
+export class PollBuilder {
+    readonly #poll: Poll.CreateStructure = <never>{};
+
+    public setQuestion(question: Poll.MediaStructure): this {
+        this.#poll.question = question;
+        return this;
+    }
+
+    public setAnswers(...answers: Array<Poll.AnswerStructure>): this {
+        this.#poll.answers = answers;
+        return this;
+    }
+
+    public addAnswer(answer: ((answer: PollAnswerBuilder) => PollAnswerBuilder) | PollAnswerBuilder): this {
+        const realAnswer = typeof answer === "function"
+            ? answer(new PollAnswerBuilder()).toJSON()
+            : answer.toJSON();
+
+        if (!Array.isArray(this.#poll.answers)) this.#poll.answers = [];
+        this.#poll.answers.push(realAnswer);
+
+        return this;
+    }
+
+    public setDuration(duration: number): this {
+        this.#poll.duration = duration;
+        return this;
+    }
+
+    public setAllowMultiselect(allowMultiselect: boolean): this {
+        this.#poll.allow_multiselect = allowMultiselect;
+        return this;
+    }
+
+    public setLayoutType(layoutType: PollLayoutType): this {
```

Comment:
This is more of a question than a request.
Should this be just `setType` or might that not be clear enough?

Reply:
In the docs it's a layout type, so i'd keep it as it is. For example for https://discord.com/developers/docs/interactions/message-components#button-object-button-structure it's type.

Reply:
I am aware, it was just about making it a bit shorter without losing its meaning.
But its good as it is